docs/msbuild/property-functions.md

```diff
@@ -379,6 +379,39 @@ Value4 = 7.0
 Value5 = True
 ```
 
+## MSBuild version-comparison functions
+
+MSBuild 16.5 and higher define several functions for comparing strings that represent versions.
```

Comment:
**define** or **defines**?

Reply:
@ghogen got a preference here?

Reply:
define ("and higher" makes it plural)

Reply:
"and higher" is shorthand for "and later versions," so "versions define". 👍🏻